chromium/debugging-rulesets.js

```diff
@@ -0,0 +1,20 @@
+/* global sendMessage */
+
+"use strict";
+
+window.onload = function() {
+  document.querySelector("form").addEventListener("submit", save_debugging_rulesets);
```

Comment:
This works fine, but it might be better to give the form an id, instead of with this queryselector, just to be defensive in-case another form is added some day.